docker/runtime/Dockerfile.pytorch.aarch64

```diff
@@ -5,5 +5,5 @@ ARG WHEEL_FILE=torch_blade*.whl
 
 ADD ./build/${WHEEL_FILE}  /install/python/
 
-RUN source /opt/venv_disc/bin/activate && \
+RUN . /opt/venv_disc/bin/activate && \
```

Comment:
For L1, maybe it's better not to use devel Docker as `BaseImage`. As we expect, the devel Docker includes some tools for development just like cmake, bazel, and runtime Docker would much smaller than the devel one, the runtime Docker just includes required software to run a compiled binary or wheel.

Reply:
We do not have a community tensorflow image for arm.  So we have to build docker image from scratch.  I think it's not relevant to this commit and can be leaved for future improvement.

Reply:
Agree with that.